examples/prescriptors/neat/README.md

```diff
@@ -0,0 +1,82 @@
+## NEAT Prescriptor
```

Comment:
Seconded on the good `README` -- thanks for taking the time to write this up. It will help our competitors (and us, in future).